distribution/lib/Standard/Base/0.0.0-dev/src/Data/Arrow_Array_Builder.enso

```diff
@@ -0,0 +1,44 @@
+import project.Any.Any
+import project.Data.Vector.Vector
+import project.Data.Numbers.Integer
+import project.Internal.Array_Like_Helpers
+import project.Errors.Common.Index_Out_Of_Bounds
+import project.Nothing.Nothing
+
+@Builtin_Type
+type Arrow_Array_Builder
+    ## PRIVATE
+
+        Sets a provided value at a given index.
+    set : Integer -> Any -> Nothing ! Index_Out_Of_Bounds
+    set self index value =
+        Arrow_Array_Builder.set_builtin self index value
```

Comment:
Normally, our builders only allow an "append" method where just set the current index and moves one further on.

Reply:
added, left `set` in the `spec` as an extension. I can remove even that if you think it is a bad idea.